python/vesin/tests/test_neighbors.py

```diff
@@ -1,28 +1,57 @@
-import ase
+import os
+
 import ase.build
+import ase.io
 import ase.neighborlist
 import numpy as np
+import pytest
 
 import vesin
 
+CURRENT_DIR = os.path.dirname(os.path.realpath(__file__))
+
 
-def test_neighbors():
-    atoms = ase.build.make_supercell(ase.build.bulk("Si", "diamond"), 3 * np.eye(3))
+@pytest.mark.parametrize("system", ["water", "diamond", "naphthalene", "carbon"])
+@pytest.mark.parametrize("quantities", ["ijSD", "ijS"])
```

Comment:
I don't see the point of this parametrization, the second one just computes a subset of the first one.

Reply:
Hmm well, given that I didn't check how the library works, I thought it was a good idea. But if you think ijS will always be correct if ijSD is, then it can be removed

Reply:
yes, everything is done at the same time, it's only a question of do we throw away this data or do we store it in some allocation